dev/com.ibm.ws.sipcontainer/src/com/ibm/ws/sip/stack/transport/sip/netty/SipUdpConnLink.java

```diff
@@ -166,7 +165,7 @@ public void run() {
 					// wait here until there's something to send
 					MessageContext messageContext = null;
 					synchronized (m_outMessages) {
-						if (m_outMessages.isEmpty() || !m_connLink.m_connected) {
+						if (m_outMessages.isEmpty()) {
```

Comment:
Waits until a message is added to the queue ( that's when `m_outMessages.notify();` is called.